homeassistant/components/opengarage/cover.py

```diff
@@ -161,23 +157,25 @@ def update(self):
 
     def _push_button(self):
         """Send commands to API."""
+        result = 0
         try:
             response = requests.get(
                 "{}/cc?dkey={}&click=1".format(self.opengarage_url, self._device_key),
                 timeout=10,
             ).json()
+            result = response["result"]
```

Comment:
Please move this down outside the try... except.

Reply:
Sorry to bother you @MartinHjelmare but this platform needs an external library right?
https://github.com/home-assistant/home-assistant/pull/15397#discussion_r207104952

Reply:
Yes, it does. This PR is just a clean up.